test-cases/scylla-operator/kubernetes-operator-upgrade.yaml

```diff
@@ -0,0 +1,13 @@
+test_duration: 300
+
+# workloads
+stress_cmd_r: cassandra-stress read no-warmup cl=QUORUM n=2010020 -schema 'keyspace=keyspace_entire_test replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=100 -pop seq=1..2010020 -log interval=5
+stress_cmd_w: cassandra-stress write no-warmup cl=QUORUM n=2010020 -schema 'keyspace=keyspace_entire_test replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=100 -pop seq=1..2010020 -log interval=5
+
+n_db_nodes: 3
+n_loaders: 1
+n_monitor_nodes: 1
+
+use_mgmt: false
```

Comment:
Why ae we running it without scylla-manager ?

Reply:
It is not used in "upgrade" jobs.